### ISSUE TITLE
"Quit fman" menu entry remains disabled

### ISSUE THREAD

**Issue by mherrmann** [fman-community/fman#328]
This is a continuation of #327. On Mac, when any (modal) dialog is open, such as the splash screen or the Command Palette, and the user clicks on the menu, the `Quit fman` entry is disabled:

![screen1_cropped](https://user-images.githubusercontent.com/1076393/36603760-21c36678-18bc-11e8-95b6-80df5d222ca2.png)

That may be justifiable because the dialog is modal. What's not justifiable however is that the menu item then _remains_ disabled even after the dialog was closed:

![screen2_cropped](https://user-images.githubusercontent.com/1076393/36603783-35f0b024-18bc-11e8-915f-d652fece7112.png)

The most likely reason for this is a [bug in Qt](https://bugreports.qt.io/browse/QTBUG-63866).

**Comment by gingerbeardman** [comment]
Interestingly the shortcut key combo <kbd>Cmd</kbd>+<kbd>Q</kbd> still works to quit fman.

**Comment by mherrmann** [comment]
Yes. That was made possible by #327. It's a really strange bug. As written above, I think the underlying cause lies in Qt. Thanks for pointing it out though @gingerbeardman! It wasn't clear above.

**Comment by gingerbeardman** [comment]
This is fixed.
